velox/docs/configs.rst

```diff
@@ -164,4 +164,27 @@ data size exceeds this limit, otherwise it spills the partition with most data.
 If the limit is zero, then the spiller always spills a previously spilled
 partition if it has any data. This is to avoid spill from a partition with a
 small amount of data which might result in generating too many small spilled
-files.
\ No newline at end of file
+files.
+
+
+Hive Connector
+-----------------------------
+
+``max_partitions_per_writers``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``integer``
+    * **Default value:** ``100``
+
+Maximum number of partitions per writer.
+
+``insert_existing_partitions_behavior``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``string``
+    * **Allowed values:** ``OVERWRITE``, ``ERROR``
+    * **Default value:** ``ERROR``
+
+Set the behavior on insert existing partitions in write protocol. ``OVERWRITE``
```

Comment:
As discussed offline, this configuration property doesn't have any impact on the worker's behavior and is used only to populate the update-mode field in the writer's result. It would be nice to clarify this, otherwise, readers will expect some logic in the worker that doesn't exist.

Reply:
Revised